priv/schema/rabbit.schema

```diff
@@ -1756,7 +1756,7 @@ end}.
 
 {validator, "byte", "Integer is not 0<i<255",
```

Comment:
This comment sounds wrong (and in particular now). Should we correct it?

Reply:
:facepalm: of course. I'll get it.

Reply:
255 is valid as well. One more sec!